### ISSUE TITLE
Flagstat email is being sent before new RNA BAM & Flagstat are being run

### ISSUE THREAD

**Issue by ihodes** [hammerlab/epidisco#60]

**Comment by ihodes** [comment]
Same for FASTQC; problem is naming of fastq-results/flagstat-results; need a has in it to force recomputation with new data.

**Comment by ihodes** [comment]
May even start earlier with the names of concat read1 and read2 steps (for fastqc).

**Comment by smondet** [comment]
So if I understand correctly you changed the dataset without renaming it?

**Comment by ihodes** [comment]
The above problem is a bug; we generate a file called fastq-summary.txt; the name doesn't change when new data is provided (even if a new dataset name is provided). 

I did add FASTQ files without changing the name of the dataset. I think requiring a unique name whenever the dataset changes is something Biokepi should handle (with a hash of the input FASTQs) rather than the user, though.

**Comment by smondet** [comment]
@ihodes no, different datasets should have different names/ids. If not, we start letting insanity in and then it becomes debt that we have to deal with forever.

**Comment by ihodes** [comment]
I agree; but Biokepi can name them for us. 

Regardless, I don't believe that's the problem here, as the summary file names don't take dataset into account.
